crates/sui-display/src/v2/parser.rs

```diff
@@ -906,6 +913,36 @@ impl<'s> Parser<'s> {
 
     fn parse_xform(&mut self) -> Result<Transform, FormatError> {
         Ok(match_token! { self.lexer;
+            Lit(_, T::Ident, _, "base64") => {
```

Comment:
Nit: Consider pulling these names up into constants

Reply:
So I didn't pull these out into constants for two reasons:

- Using constants in match patterns leads to quite dangerous foot guns. If you forget to import or define the constant (or there's a typo in the constant name) it turns into a catch-all pattern, and it can be a very difficult thing to spot.
- I find it's only useful to put something in a constant if I'm likely to change the constant's value without changing the constant's name, i.e. I don't see how I would get much value out of defining a constant `const BASE64: &'static str = "base64";`, which it would seem is how most of these constants would look, unless I'm misunderstanding?